examples/bubbling.rs

```diff
@@ -0,0 +1,90 @@
+use bevy::prelude::*;
+use bevy_mod_picking::prelude::*;
+
+fn main() {
+    App::new()
+        .add_plugins(DefaultPlugins)
+        .add_plugins(DefaultPickingPlugins)
+        .add_plugin(backends::RaycastPlugin)
+        .add_plugin(DebugEventsPlugin)
+        .add_startup_system(setup)
+        .add_system(handle_events)
+        .run();
+}
+
+struct DeleteMe(Entity);
+impl EventFrom for DeleteMe {
+    fn new(event_data: &mut EventData<impl IsPointerEvent>) -> Self {
```

Comment:
These examples will be much clearer if you avoid using `Self` in these impl blocks.